update_docker_images_txt.py

```diff
@@ -0,0 +1,74 @@
+"""Update ./docker_images.txt."""
+
+import argparse
+import logging
+
+
+def main() -> None:
+    """Prep and execute Condor job.
+
+    Make scratch directory and condor file.
+    """
+    parser = argparse.ArgumentParser(
+        description=("Update the ./docker_images.txt"),
+        formatter_class=argparse.ArgumentDefaultsHelpFormatter,
+    )
+
+    parser.add_argument(
+        "--action",
+        required=True,
+        choices=["add", "remove"],
+        help="What do you want to do with the docker images?",
+    )
+    parser.add_argument(
+        "--docker-tag",
+        required=True,
+        help="The docker image tag",
+    )
+    parser.add_argument(
+        "--dest-dir",
+        required=True,
+        help="The destination directory, eg: realtime",
+    )
+    parser.add_argument(
+        "--remove-docker-repo",
+        default=False,
+        action="store_true",
+        help="whether to remove the docker image's repo when inserting to CVMFS dir",
+    )
+
+    args = parser.parse_args()
+    for arg, val in vars(args).items():
+        logging.warning(f"{arg}: {val}")
+
+    # TAG  = "icecube/skymap_scanner:3"
+    # LINE = "docker://icecube/skymap_scanner:3 realtime/skymap_scanner:3"
+
+    if args.remove_docker_repo:
+        dest_file = args.docker_tag.split("/", max_split=1)[1]
+    else:
+        dest_file = args.docker_tag
+
+    line_to_add = f"docker://{args.docker_tag} {args.dest_dir}/{dest_file}"
+
+    def negated() -> str:
+        return f"- {line_to_add}"
+
+    with open("./docker_images.txt", "w") as f:
+        lines = [ln.strip() for ln in f.readlines()]  # rm each trailing '\n'
```

Comment:
You're going to want to open the file first in read mode, get the lines, then open it again for writing.

Technically you could open it read/write and seek back to the beginning, but that seems error-prone when it's unnecessary.

Reply:
Aha, very C-like